.eleventy.js

```diff
@@ -2,11 +2,11 @@ const fs = require('fs');
 
 // Plugins
 const eleventyNavigationPlugin = require('@11ty/eleventy-navigation');
-const sitemap = require('@quasibit/eleventy-plugin-sitemap');
+const build = require('./src/_data/build');
```

Comment:
Can we omit this if we're not doing env-specific minification?

Reply:
It's being used for the sitemap plugin.

```
eleventyConfig.addPlugin(sitemap, {
    sitemap: {
      hostname: build.isProduction ? 'https://ceph.io' : 'https://develop.ceph.io',
    },
  });
```

Reply:
Ah, my bad. Was just confused by the shift in these diffs. Makes sense 👍  Thanks for clarifying

Reply:
I should probably have mentioned that I sorted the plugins into alphabetical order 🤪